src/appleseed/renderer/modeling/camera/camera.cpp

```diff
@@ -295,6 +305,44 @@ double Camera::extract_near_z() const
     return near_z;
 }
 
+namespace
+{   
+    float map_sample_to_linear_curve(const float a, const float b, const float x)
+    {
+        float constant;
+        // opening
+        if (a > 0)
```

Comment:
Type all floating point literals appropriately, i.e. 0.0f for float and 0.0 for double.